Makefile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2015 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+.PHONY: all format clean master manager fuzzer executor
+
+all: master manager fuzzer executor
```

Comment:
The folder is created automatically right now, but I added `mkdir` anyway, just in case.